packages/portal-proto/tsconfig.json

```diff
@@ -21,13 +21,20 @@
       "@/utils/*": ["src/utils/*"],
       "@/hooks/*": ["src/hooks/*"]
     },
-    "incremental": true
+    "incremental": true,
+    "strictNullChecks": false,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ]
   },
   "include": [
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
-    "src/components/ItemTypes.js"
+    "src/components/ItemTypes.js",
+    ".next/types/**/*.ts"
```

Comment:
This seems off to me. The `.next` directory is output directory for nextjs builds.  If you have a freshly cloned repo (or run `npm run clean`), then the `.next` directory will not exists and typescript won't be able to import typings from it.

Reply:
This was auto generated by the next js when running `next dev` for the first time after the upgrade. I had noticed that even if a user doesn't have a `.next` director running `npm run dev` for the first time will create one for them and populate it with files needed.
<img width="274" alt="Screenshot 2024-02-09 at 11 08 18 AM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/101295912/170d1ed3-bc16-4ff1-b0f6-7b080d7f2ff8">

Reply:
I don't understand why nextjs does that, but it doesn't seem to hurt anything then we can ignore it for now.

Have you verified that it still works in production mode?   You should be able to do this by running the prod server instead of the dev server:
```
npm run build
npm run start
```